eng/Versions.props

```diff
@@ -20,10 +20,18 @@
     <XmlDocPackage>Microsoft.Private.Intellisense</XmlDocPackage>
     <XmlDocPackageVersion>2.0.0-preview3-26209-0</XmlDocPackageVersion>
   </PropertyGroup>
+
+  <PropertyGroup>
+    <NetStandardLibraryPackage>netstandard.library</NetStandardLibraryPackage>
+  </PropertyGroup>
+  
   <PropertyGroup>
     <MicrosoftDotNetApiCompatPackageVersion>1.0.0-beta.19061.6</MicrosoftDotNetApiCompatPackageVersion>
     <MicrosoftDotNetBuildTasksPackagingPackageVersion>1.0.0-beta.19061.6</MicrosoftDotNetBuildTasksPackagingPackageVersion>
     <MicrosoftDotNetCodeAnalysisPackageVersion>1.0.0-beta.19061.6</MicrosoftDotNetCodeAnalysisPackageVersion>
     <MicrosoftDotNetGenAPIPackageVersion>1.0.0-beta.19061.6</MicrosoftDotNetGenAPIPackageVersion>
+
+    <!-- This is the LKG version of NetStandard.Library from myget.org, before we bumped the TFM to netstandard2.1. Please don't edit this version -->
+    <NetStandardLibraryPackageVersion>2.1.0-preview1-27119-01</NetStandardLibraryPackageVersion>
```

Comment:
@ericstj do you think this is a good version of the package to depend on? Or should I go with the latest 2.0 stable version, 2.0.3 (which is from April 2018)?

Reply:
You should use latest stable.